Sophiah123/statuscorrectionsuidoc (#11462)
DataDog/documentation: content/en/monitors/service_level_objectives/_index.md

Status corrections allow you to identify specific time periods, such as planned maintenance, that an SLO should ignore from its calculation of status and error budget. Data during excluded time periods is not taken into account. A public beta for corrections is available through the [SLO status corrections API][12] and [Terraform][13].

## SLO status corrections

> Status corrections allow you to identify specific time periods, such as planned maintenance, that an SLO should ignore from its calculation of status and error budget. 

Reviewer: ```suggestion
Status corrections allow you to specify time periods such as planned maintenance so that SLOs exclude the time from status and error budget calculations.
```

> When you create a correction window for a SLO, the time period you specify will be removed from the SLO’s calculation. 

Reviewer: ```suggestion
When you create a correction window for an SLO, the time period you specified is removed from the SLO’s calculation. 
```

> - For monitor-based SLOs, this means that any time that occurred during the correction window is no longer counted

Reviewer: ```suggestion
- For monitor-based SLOs, time in the correction window is not counted.
```

> - For metric-based SLOs, this similarly means that all the good and bad events that occurred during the correction are not counted

Reviewer: ```suggestion
- For metric-based SLOs, all good and bad events in the correction window are not counted.
```

> A public beta for corrections is available through the [SLO status corrections API][12], [Terraform][13], and the UI.

Reviewer: ```suggestion
<div class="alert alert-warning">
Status corrections are in public beta. See the [SLO status corrections API][12], [Terraform][13], and the UI.
</div>
```

> #### Access SLO status corrections in the UI 

Reviewer: ```suggestion
#### Access in the UI 
```

To access SLO status corrections in the UI:

> 1. Create a new SLO or edit an existing one
> 2. Navigate to an SLO’s detailed side panel view
> 3. Under the gear icon, select “Correct Status” to access the Status Corrections creation modal
> 4. Select the Time Correction Window, Correction Type, and add Notes
> 5. Click “Apply Correction”

Reviewer: ```suggestion
1. Create a new SLO or edit an existing one.
2. Navigate to an SLO’s detailed side panel view.
3. Under the gear icon, select **Correct Status** to access the **Status Corrections** creation modal.
4. Select the **Time Correction Window**, **Correction Type**, and add **Notes**.
5. Click **Apply Correction**.
```

> To view, edit, and delete existing status corrections, click on the “Corrections” tab at the top of an SLO’s detailed side panel view. 

Reviewer: ```suggestion
To view, edit, and delete existing status corrections, click on the **Corrections** tab at the top of an SLO’s detailed side panel view. 
```
